File: SoftUni Lessons/Python Development/Python Basics April 2019/Lessons and Problems/04 - Conditional Statements/04. Number 1...9 to Text .py
"""
Conditional Statements - Lab
Проверка: https://judge.softuni.bg/Contests/Practice/Index/1012#0

04. Number 1...9 to Text

Условие:
Да се напише програма, която чете цяло число в диапазона [1…9],
въведено от потребителя и го изписва с думи на английски език.
Ако числото е извън диапазона, програмата изписва "number too big".
Примерен вход и изход
5	five		1	one		9	nine		10	number too big
"""

num = int(input())

if num == 1:
    print("one")
elif num == 2:
    print("two")
elif num == 3:
    print("three")
elif num == 4:
    print("four")
elif num == 5:
    print("five")
elif num == 6:
    print("six")
elif num == 7:
    print("seven")
elif num == 8:
    print("eight")
elif num == 9:
    print("nine")
else:
    print("number too big")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/07. ПРОВЕРКИ/04. Number 1...9 to Text.js
// There are two ways of reading the Input for this tasks:
// This is the new way:
function numbersOneToNine(number) {
    // Look.... I know it looks weird, but I wanted to try using objects :)
    numbers = {
        1: "one",
        2: "two",
        3: "three",
        4: "four",
        5: "five",
        6: "six",
        7: "seven",
        8: "eight",
        9: "nine" 
    };
    if (number in numbers) {
        console.log(numbers[number]);
    } else if (false) {
        // This code is unreachable... jsut to use the esle if :D
    } else {
        console.log("number too big");
    }
}

numbersOneToNine(3); // Expected output: three
numbersOneToNine(1); // Expected output: one
numbersOneToNine(9); // Expected output: nine
numbersOneToNine(69); // Expected output: number too big




// -----------------------------------------------------------------------//
// This is the old way of reading the input:
function numbersOneToNineOld(input) {
    number = Number(input.shift());
    // Look.... I know it looks weird, but I wanted to try using objects :)
    numbers = {
        1: "one",
        2: "two",
        3: "three",
        4: "four",
        5: "five",
        6: "six",
        7: "seven",
        8: "eight",
        9: "nine" 
    };
    if (number in numbers) {
        console.log(numbers[number]);
    }
    else if (false) {
        // This code is unreachable... jsut to use the esle if :D
    }
    else {
        console.log("number too big");
    }
}

numbersOneToNineOld([3]); // Expected output: three
numbersOneToNineOld([1]); // Expected output: one
numbersOneToNineOld([9]); // Expected output: nine
numbersOneToNineOld([69]); // Expected output: number too big
